### DISCUSSION TITLE
Lodestar Standup 2024-12 (March 26, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6577] [Standups]
### Meeting Info
[Mar 26, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/mar-26-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Planning Items
  - v1.18.0 targeted PR updates
  - Yamux testing
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by twoeths** [comment]
Need to discuss blockers for increasing target peers to 100, see https://github.com/ChainSafe/lodestar/pull/6556#issuecomment-2017692440

TLDR:
- There's likely memory issue that only happen on mainnet nodes with mainnet validators
- Devops to download network heap snapshot for instances >= 2GB
- Devops need to configure target peers to 100, find instance with memory leak and download heap snapshot there. May need to do it before a new deployment to minimize the impact
- Tuyen to investigate if increasing peers to 100 may cause a delay when publish. Check `floodPublish` option of other clients

**Comment by philknows** [comment]
Including this discussion with a reference on some censorship data: https://github.com/ChainSafe/lodestar/pull/6568#issuecomment-2020503894

**Comment by matthewkeil** [comment]
Supporting all versions of Node that is still in LTS/Maintenance.

**Comment by philknows** [comment]
Closing for https://github.com/ChainSafe/lodestar/discussions/6615
